### ISSUE TITLE
add title screen UI to manage/restore backups

### ISSUE THREAD

**Issue by pcal43** [pcal43/fastback#55]

**Comment by Felix14-v2** [comment]
I agree, just make this button open custom UI to manage world snapshots easily
![2022-09-18_22 29 33](https://user-images.githubusercontent.com/75726196/190925039-052b31a9-4dc5-4f53-8d13-5bfcad980cb0.png)
